app/react-native/src/preview/components/StoryListView/index.js

```diff
@@ -57,50 +60,90 @@ export default class StoryListView extends Component {
     stories.removeListener(Events.STORY_ADDED, this.storyAddedHandler);
   }
 
+  buildStoriesTree=(stories)=>{
+    const { selectedKind, selectedStory } = this.props;
+
+    let hashmap={};
+    let result=[];
+    stories.forEach(story=>{
+      let components=story.kind.split("/");
```

Comment:
this should leverage `hierarchySeparator`/`hierarchyRootSeparator`

Reply:
I didn't know about this config. What is the difference between them?

Reply:
The separators are available through https://github.com/storybooks/storybook/blob/31b8243db4760760b8d71334805fa910dc19333c/lib/client-api/src/client_api.js#L90

https://storybook.js.org/docs/configurations/options-parameter/ has some documentation on the two parameters. One is to identify elements at the root, the other is within the tree.